blueprints/app/files/ember-cli-build.js

```diff
@@ -20,5 +20,10 @@ module.exports = function(defaults) {
   // please specify an object with the list of modules as keys
   // along with the exports of each module as its value.
 
+  <% if (embroider) { %>
```

Comment:
This should have a fixture test now, as I believe you have some unintended whitespace now.

Reply:
Updated with a fixture test for this